app-text/rpl/rpl-1.15.5.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_11 )
+inherit distutils-r1
+
+DESCRIPTION="Intelligent recursive search/replace utility"
+HOMEPAGE="https://rpl.sourceforge.net/
+	https://github.com/rrthomas/rpl"
+SRC_URI="
+	https://github.com/rrthomas/rpl/archive/v${PV}.tar.gz
+		-> ${P}.gh.tar.gz
+"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86"
```

Comment:
I've dropped everything but ~amd64 as it needs to go through rekeywording, see https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades. Thanks!